bin/mastarm

```diff
@@ -9,7 +9,7 @@ const util = require('../lib/util')
 commander
   .version(require('../package.json').version)
   .option('-c, --config <path>', 'Path to configuration files.', path.join(process.cwd(), '/configurations/default'))
-  .option('-e, --env <environment>', 'Environment to use.', process.env.NODE_ENV || 'development')
+  .option('-e, --env <environment>', 'Environment to use.')
```

Comment:
This breaks the way tests are run in the scenario editor project.  It is less characters to have the -e flag and this option would work cross-platform, so maybe it's a good idea to keep it this way.